src/dlgprefreplaygaindlg.ui

```diff
@@ -202,7 +248,20 @@
       <item row="4" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
-         <string>It may take a few seconds after track analysis is finished for a newly calculated ReplayGain to be applied. To avoid abrupt volume changes, Mixxx does not change the ReplayGain of playing tracks.</string>
+         <string>It may take a few seconds after track analysis is finished for a newly calculated ReplayGain to be applied. To avoid abrupt volume changes, Mixxx does not apply the ReplayGain to playing tracks. </string>
+        </property>
+        <property name="wordWrap">
+         <bool>true</bool>
+        </property>
+       </widget>
+      </item>
+      <item row="0" column="0">
+       <widget class="QLabel" name="label_8">
+        <property name="text">
+         <string>The normal average ReplayGain playback level is -14 dB. This offers enough headroom for the dynamic peaks of almost all tracks. However it might be not enough for mixing or adding effects without clipping or it might sound to quiet for modern pop music that is normalized to 0 db and compressed to an average playback level of -7 dB or more.  So you may tweak the Initial Boost to solve these issues.</string>
```

Comment:
I'm able to understand this sentene only because I have read the bug report to this PR. The sentence is to complicated and contains to much information at this point. You only want to say that if a user experiences clipping during normal usage he can adjust the "initial boost" to get rid of it.  More information about the topic can go into the manual. Adding a hint about the manual would also be fine.